optuna/multi_objective/visualization/_pareto_front.py

```diff
@@ -112,20 +121,47 @@ def _get_pareto_front_2d(
         point_colors += ["red"] * len(non_pareto_trials)
         trials += non_pareto_trials
 
+    if axis_order is None:
+        axis_order = list(range(2))
+    else:
+        if len(axis_order) != 2:
+            raise ValueError(
+                f"Size of `axis_order` {axis_order}. Expect: 2, Actual: {len(axis_order)}"
```

Comment:
Again a detail but we usually place a period.
```suggestion
                f"Size of `axis_order` {axis_order}. Expect: 2, Actual: {len(axis_order)}."
```